useful_resources/sample_code/README.md

```diff
@@ -12,12 +12,15 @@ Please also add your new script to the table below.
 
 # Existing Scripts
 
-| File Name                   | Brief Description                                                              | Connectors Used       | Written For Parsons Version |
+| File Name                       | Brief Description                                                              | Connectors Used       | Written For Parsons Version |
 | --------------------------- | ------------------------------------------------------------------------------ | --------------------- | --------------------------- |
-| apply_activist_code.py      | Gets activist codes stored in Redshift and applies to users in Van             | Redshift, VAN         | unknown                     |
-| s3_to_redshift.py           | Moves files from S3 to Redshift                                                | Redshift, S3          | unknown                     |
-| s3_to_s3.py                 | Get files from vendor s3 bucket and moves to own S3 bucket                     | S3                    | unknown                     |
-| update_user_in_actionkit.py | Adds a voterbase_id (the Targetsmart ID) to users in ActionKit                 | Redshift, ActionKit   | unknown                     |
-| zoom_to_van.py              | Adds Zoom attendees to VAN and applies an activist code                        | Zoom, VAN             | 0.15.0                      |
-| ngpvan_sample_list.py       | Creates a new saved list from a random sample of an existing saved list in VAN | VAN                   | unknown                     |
-| actblue_to_google_sheets.py | Get information about contributions from ActBlue and put in a new Google Sheet | ActBlue, GoogleSheets | 0.18.0                      |
+| actblue_to_google_sheets.py     | Get information about contributions from ActBlue and put in a new Google Sheet | ActBlue, GoogleSheets | 0.18.0                       |
+| apply_activist_code.py          | Gets activist codes stored in Redshift and applies to users in Van             | Redshift, VAN         | unknown                     |
+| civis_job_status_slack_alert.py | Posts Civis job and workflow status alerts in Slack                            | Slack                 | unknown                     |
+| mysql_to_googlesheets.py        | Queries a MySQL database and saves the results to a Google Sheet               | GoogleSheets, MySQL   | unknown                     |
+| ngpvan_sample_list.py           | Creates a new saved list from a random sample of an existing saved list in VAN | VAN                   | unknown                     |
+| opt_outs_everyaction.py         | Opts out phone numbers in EveryAction from a Redshift table                    | Redshift, VAN         | unknown                     |
```

Comment:
We might be getting rid of this sample code table at some point, as we switch to keeping info in our docs, so this not hugely important, but: is there a Parsons version number associated with this script?

Reply:
I don't have a version number, the script has been running fine through several iterations of Parsons